seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/multitablesink/MultiTableSinkWriter.java

```diff
@@ -75,6 +80,7 @@ public MultiTableSinkWriter(
             BlockingQueue<SeaTunnelRow> queue = new LinkedBlockingQueue<>(1024);
             Map<String, SinkWriter<SeaTunnelRow, ?, ?>> tableIdWriterMap = new HashMap<>();
             Map<SinkIdentifier, SinkWriter<SeaTunnelRow, ?, ?>> sinkIdentifierMap = new HashMap<>();
+            Map<SinkIdentifier, SinkWriter.Context> sinkContextIdentifierMap = new HashMap<>();
```

Comment:
This variable useless?

Reply:
Yes, this may be a useless plan. I have already deleted it